### PR TITLE
Minor doc tweaks.

### PR DESCRIPTION
**Pull request by obrien-j**
1. Remove duplicate browserstack ref
2. Slightly more specific wording for Snyk, to distinguish between static code analysis and detection/notification of vulnerable dependencies.

### CODE REVIEW

**Comment by SupeDeDupe** [comment]
Is there an issue for this PR? Can you put closes #issuenumber in the PR description? If there's not issue for it, then you can create one to do this. It's just how we've been doing things to keep track of the purpose of a PR.

File: doc/README.md [hunk 1]
```diff
@@ -6,7 +6,6 @@
 - [AirTable](/doc/AIRTABLE.md)
 - [Heroku](#heroku)
 - [CircleCI](#circleci)
-- [BrowserStack](#browserstack)
```
Comment (smcmurtry) [inline doc/README.md:9]: Was this an accident? We still have a section in this doc for browserstack

Comment (obrien-j) [reply doc/README.md:9]: Line #9 and #11 == duplicates. Hence the removal.

**Comment by obrien-j** [comment]
@SupeDeDupe No issue created, as I was just passing by whilst bored on parental leave. I'll leave it up to y'all to patch in with your existing process. Just figured I'd point it out.

File: doc/README.md (hunk 1 above)
Comment (smcmurtry) [reply doc/README.md:9]: Oh of course, good catch!

**Review by smcmurtry — APPROVED** [review]
Thanks John!

**Pull request merged**